projects/peerdas-metrics.md

```diff
@@ -0,0 +1,78 @@
+# PeerDAS Metrics
+
+Improving PeerDAS testing efficiency via metrics implementation and visualization. 
+
+## Motivation
+
+PeerDAS is now entering its testing phase, with most CL clients actively participating. Identifying and utilizing specific metrics is crucial for evaluating efficiency, troubleshooting, and optimizing the protocol. Before PeerDAS is deployed on the mainnet, detailed testing is important to keep the Ethereum Protocol secure and efficient. The project aims to analyze and present these results in an accessible format for core developers and the research team.
+
+## Project description
+
+PeerDAS Metrics focuses on identifying the most crucial metrics, integrating them into CL clients (Teku, Grandine), and developing visual representations to help teams use these metrics effectively. Additionally, the project will present the analysis of these metrics, providing valuable insights for the research team.
+
+## Specification
+
+The project is executed through these specific steps:
+
+- Monitor and visualize the initial set of metrics using Kurtosis and Grafana.
+
+- Review the initial metrics, identify additional ones that could be beneficial.
```

Comment:
Lighthouse integrated metrics for peerdas development in its branch, Andrew listed it in the r&d discord message. My idea was to get inspiration from that and apply it in other clients. If so, the actual list of metrics or at least vaguely categorized should be here